src/NzbDrone.Core/Indexers/Definitions/BeyondHD.cs

```diff
@@ -274,6 +276,18 @@ public IList<ReleaseInfo> ParseResponse(IndexerResponse indexerResponse)
                     flags.Add(IndexerFlag.Internal);
                 }
 
+                // BHD API is broken on limited filter
```

Comment:
```suggestion
                // ToDo: Remove when BHD Fixes API. BHD API is broken on limited filter
```

any ETA from staff when they may have it fixed up?

Reply:
No ETA on their side for now.